Add FloatingComponent prop (#162)
jeremybarbet/react-native-modalize: docs/PROPSMETHODS.md

| -------- | -------- |
| object   | No       |

### `FloatingComponent`

> A floating component that will overlay Modalize completely, independent of scrolling. It requires zIndex child with absolute positioning. Check [Flatlist example](../examples).

Reviewer: Not sure if you wanted to link to the example git repository or page on the documentation.

```suggestion
A floating component inside the modal wrapper that will be independent of scrolling. It requires `zIndex` child with absolute positioning. Check out the [Flatlist example](https://github.com/jeremybarbet/react-native-modalize/blob/master/examples/expo/src/components/modals/FlatList.js).
```

| Type     | Required |
| -------- | -------- |

> | node      | No       |

Reviewer: ```suggestion
| node     | No       |
```

> | any      | No       |

Reviewer: ```suggestion
| node      | No       |
```
